Fix Bug with Alias Account

ThreeAndTwo opened the pull request:
1. Bug Fix: Alias Account Overwriting
2. Code Improvement: Use of Constants

Commit:
Fix Bug with Alias Account

1. Bug Fix: Alias Account Overwriting
2. Code Improvement: Use of Constants

RonTuretzky commented:
@uri-lightblocks thought?

uri-lightblocks commented on internal/operatorcli/operator_key.go:
```diff
@@ -0,0 +1,100 @@
+	aliasECDSA    *ecdsa.PrivateKey
+}
+
+func NewMangeKey(
```
passPhrase should be the same for all keys

uri-lightblocks replied:
rename function name (and other references from mange to manage - add `a`)

uri-lightblocks commented on internal/operatorcli/operator_key.go:
```diff
@@ -0,0 +1,100 @@
+	aliasECDSAPassphrase   string
+}
+
+type priKeys struct {
```
rename to privateKeys

uri-lightblocks replied:
add bls private key

uri-lightblocks commented on internal/operatorcli/operator_key.go:
```diff
@@ -0,0 +1,100 @@
+	}
+}
+
+func (m *mangeKey) getPrivateKeys() (priKeys, error) {
```
add reading the bls private key

uri-lightblocks commented on internal/operatorcli/operator_key.go:
```diff
@@ -0,0 +1,100 @@
+func NewMangeKey(
+	keyStorePath string,
+	operatorECDSAFile string,
+	operatorECDSPassphrase string,
```
missing `A` after `ECDS`

uri-lightblocks commented on internal/operatorcli/operator_key.go:
```diff
@@ -0,0 +1,100 @@
+		var ecdsaPair *ecdsa.PrivateKey
+		// generate
+		if m.aliasECDSA != "" {
+			if m.aliasECDSA[:2] == "0x" || m.aliasECDSA[:2] == "0X" {
```
check the length of aliasECDSA

Commit:
Enhancements and Standardizations in Cryptographic Key Handling

1. fixed spelling errors
2. added logic to read BLS key
3. standardize encryption/decryption using passPhrase

ThreeAndTwo commented:
The code has been updated according to the review comments, and a re-review is requested.

uri-lightblocks commented on internal/operatorcli/operator_cli.go:
```diff
@@ -478,30 +478,14 @@ func RunEOChainSetAlias(c *cli.Context) error {
-	// Save the private key to a file
-	if err = eigensdkecdsa.WriteKey(filepath.Join(keyStorePath, "ecdsaAliasedEncryptedWallet.json"), ecdsaPair, passphrase); err != nil {
-		return cli.Exit(fmt.Sprintf("Error writing the ecdsaAliasedEncryptedWallet.json file %v", err), 1)
+	_mangeKey := NewMangeKey(keyStorePath, flag.EncryptedOperatorECDSAFile, passphrase,
```
I didn't see where you encrypt the alias private key

RonTuretzky commented:
This has been resolved upstream by the separation of alias generation and declaration.
